test/Controller/AboutPeopleControllerTest.php

```diff
@@ -184,6 +184,7 @@ public function it_has_metadata()
     protected function getUrl() : string
     {
         $this->mockPeopleApi(['leadership', 'senior-editor']);
+        $this->mockPeopleApi(['']);
```

Comment:
Is this up to date? It seems `mockPeopleApi()` will only mock calls to `/people` but now the new call being introduced by this PR is `/people/6d42f4fe` so it's not covered by this anyway?

Reply:
We don't need `$this->mockPeopleApi(['']);` any more. It was left over from a previous approach. Removing.

Reply:
> the new call being introduced by this PR is /people/6d42f4fe so it's not covered by this anyway?

Correct. This is not covered.